src/client_backend/triton_c_api/triton_loader.cc

```diff
@@ -1049,21 +1049,26 @@ TritonLoader::GetOutputs(
     std::string data_type{datatype};
     std::vector<uint8_t> data_copy;
 
-    if (memory_type == TRITONSERVER_MEMORY_GPU) {
+    if (memory_type == TRITONSERVER_MEMORY_CPU ||
```

Comment:
If GPU is specified and TRITON_ENABLE_GPU is not defined, what will happen? Will an error be caught elsewhere in the code?

Reply:
The code would already have terminated before this part:

https://github.com/triton-inference-server/perf_analyzer/pull/254/files#diff-027378f2ac1c0f64b228c6290dd0e5aff43fd12edbfb383b5c6cd3c9f7303eceR1187-R1195

Reply:
Much cleaner, great work!